Dockerfile

```diff
@@ -5,4 +5,4 @@ LABEL authors="Phil Ewels" \
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 RUN conda env export --name nf-core-methylseq-1.4dev > nf-core-methylseq-1.4dev.yml
```

Comment:
```suggestion
RUN conda env export --name nf-core-methylseq-1.4 > nf-core-methylseq-1.4.yml
```

Reply:
Ah, well spotted! Yes we should add that!!